[2024-12-17T06:54:07.717+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-17T06:54:07.808+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_company_info_to_dwh.Staging_to_Datawarehouse manual__2024-12-17T06:50:22.462125+00:00 [queued]>
[2024-12-17T06:54:07.870+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_company_info_to_dwh.Staging_to_Datawarehouse manual__2024-12-17T06:50:22.462125+00:00 [queued]>
[2024-12-17T06:54:07.871+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-17T06:54:07.901+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): Staging_to_Datawarehouse> on 2024-12-17 06:50:22.462125+00:00
[2024-12-17T06:54:07.911+0000] {logging_mixin.py:190} WARNING - /usr/local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1344) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-17T06:54:07.911+0000] {standard_task_runner.py:72} INFO - Started process 1346 to run task
[2024-12-17T06:54:07.912+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'load_company_info_to_dwh', 'Staging_to_Datawarehouse', 'manual__2024-12-17T06:50:22.462125+00:00', '--job-id', '439', '--raw', '--subdir', 'DAGS_FOLDER/company_info_etl.py', '--cfg-path', '/tmp/tmp89gln9u2']
[2024-12-17T06:54:07.915+0000] {standard_task_runner.py:105} INFO - Job 439: Subtask Staging_to_Datawarehouse
[2024-12-17T06:54:07.999+0000] {task_command.py:467} INFO - Running <TaskInstance: load_company_info_to_dwh.Staging_to_Datawarehouse manual__2024-12-17T06:50:22.462125+00:00 [running]> on host 8e02006e0a25
[2024-12-17T06:54:08.317+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflowDWH' AIRFLOW_CTX_DAG_ID='load_company_info_to_dwh' AIRFLOW_CTX_TASK_ID='Staging_to_Datawarehouse' AIRFLOW_CTX_EXECUTION_DATE='2024-12-17T06:50:22.462125+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-17T06:50:22.462125+00:00'
[2024-12-17T06:54:08.319+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-17T06:54:08.349+0000] {base.py:84} INFO - Retrieving connection 'load_to_staging'
[2024-12-17T06:54:08.460+0000] {base.py:84} INFO - Retrieving connection 'load_to_dwh'
[2024-12-17T06:54:08.744+0000] {logging_mixin.py:190} INFO - ETL process completed successfully.
[2024-12-17T06:54:08.745+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-17T06:54:08.771+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-17T06:54:08.771+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=load_company_info_to_dwh, task_id=Staging_to_Datawarehouse, run_id=manual__2024-12-17T06:50:22.462125+00:00, execution_date=20241217T065022, start_date=20241217T065407, end_date=20241217T065408
[2024-12-17T06:54:08.809+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-17T06:54:08.883+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-17T06:54:08.888+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
